Resources/config/form-type.xml

```diff
@@ -9,5 +9,13 @@
             <tag name="form.type" alias="cmf_core_checkbox_url_label" />
             <argument type="service" id="router" />
         </service>
+
+        <service id="cmf_core.form.extension.publishable" abstract="true" class="Symfony\Cmf\Bundle\CoreBundle\Form\Extension\PublishableExtension">
+            <!-- <argument /> Extended type fqcn -->
```

Comment:
This can be avoided by using a special syntax (see http://symfony.com/doc/current/service_container/parent_services.html#overriding-parent-dependencies).

However, why don't we use a config setting in `cmf_core` to configure this argument?